hs_model_program/static/css/model_program.css

```diff
@@ -0,0 +1,56 @@
+
+.mpElement{
+    font-weight: bold;
+    padding: 2px 2px 2px 2px;
+}
+
+a.mpElement::after {
+    display:block;
```

Comment:
Fixed spacing between colons and values.  817e36322f124c70d4a82767b6ff9cad46ef851b